packages/web3/CHANGELOG.md

```diff
@@ -381,10 +381,50 @@ Documentation:
 
 -   `getName` reverse resolution
 
-## [Unreleased]
+
+## [4.11.0]
+
+### Fixed
+
+#### web3-eth-abi
+
+-   fix encodedata in EIP-712 (#7095)
+
+#### web3-utils
+
+-   `_sendPendingRequests` will catch unhandled errors from `_sendToSocket` (#6968)
+
+#### web3-eth
+
+-   Fixed geth issue when running a new instance, transactions will index when there are no blocks created (#7098)
+
+### Changed
+
+#### web3-eth-accounts
+
+- baseTransaction method updated (#7095)
+
+#### web3-providers-ws
+
+-   Update dependancies (#7109)
+
+#### web3-rpc-providers
```

Comment:
web3-rpc-providers is not re exported via web3 pkg so this should not be mentioned under web3 changelog,